base/deprecated.jl

```diff
@@ -1240,6 +1240,14 @@ end
 
 @deprecate ismatch(r::Regex, s::AbstractString) contains(s, r)
 
+function ismatch(r::Regex, s::AbstractString, offset::Integer)
```

Comment:
Wouldn't `@deprecate ismatch(r::Regex, s::AbstractString, offset::Integer) contains(SubString(s, offset+1), r)` be enough?

Reply:
it would :). Fixing